frontend/src/components/TaskRelationTable.tsx

```diff
@@ -174,13 +173,12 @@ const relationTable: (def: RelationTableDef) => FC<RelationTableProps> = ({
           itemSize={(idx) => rowHeights[idx] ?? 0}
           itemCount={tasks.length}
           height={Math.min(height, listHeight)}
-          width={editMode ? '105%' : '100%'}
+          width="105%"
         >
           {({ index, style }) => {
-            // eslint-disable-next-line @typescript-eslint/no-unused-vars
             const { height: _, width, ...rest } = style;
```

Comment:
Admittedly its a bit sad but it turned out in the user testing that users would instinctively reach for the task columns / try to click on them when attempting to edit the task relations, instead of noticing the edit toggle feature